config-model/src/main/java/com/yahoo/vespa/model/content/ContentSearchCluster.java

```diff
@@ -357,6 +357,18 @@ public void handleRedundancy(Redundancy redundancy) {
         return indexedDocTypes;
     }
 
+    public List<NewDocumentType> getDocumentTypesWithStoreOnly() {
+        List<NewDocumentType> indexedDocTypes = new ArrayList<>();
```

Comment:
Strange variabble type for a store-only set ;)

Reply:
😛